tests/ut/test_hostif_ut.py

```diff
@@ -20,20 +20,26 @@ def sai_hostif_obj(npu):
     return hostif_oid
 
 
+@pytest.fixture(scope="function")
+def hostif_dataplane(request, npu):
+    hostifs = {
+        "100": "Ethernet0"
+    }
+    request.addfinalizer(npu.hostif_dataplane_stop)
```

Comment:
In all other places we use `yield` for the teardown. If possible, I'd prefer to follow the same approach across the TCs.

Reply:
fixed.